### ISSUE TITLE
False positive: `PYI015` does not allow assigning `typing_extensions.TypeVar`

### ISSUE THREAD

**Issue by bryanforbes** [charliermarsh/ruff#3950]
Ruff version: 0.0.261

Given the following file `test.pyi`:

```python
from typing_extensions import TypeVar

_T = TypeVar('_T', infer_variance=True)
```

And the following invocation:

```sh
ruff --select PYI015 --isolated --no-cache test.pyi
```

Ruff produces the following error:

```
test.pyi:3:6: PYI015 [*] Only simple default values allowed for assignments
```

[PEP-695](https://peps.python.org/pep-0695/) and [PEP-696](https://peps.python.org/pep-0696/) are under consideration and add new parameters to `TypeVar()`. `typing_extensions` has added a `TypeVar` class in order to test out these new features before they are accepted. `flake8-pyi` does not raise `Y015` for the above.

**Comment by charliermarsh** [comment]
It looks like we allow `from typing import TypeVar` here but not `from typing_extensions import TypeVar`. I think it's just an oversight. Will double check.

**Comment by bryanforbes** [comment]
It looks like `flake8-pyi` checks for both `typing` and `typing_extensions` here: https://github.com/PyCQA/flake8-pyi/blob/main/pyi.py#L354
